tests/installation/bootloader.pm

```diff
@@ -233,6 +270,14 @@ sub run() {
     send_key "ret";
 }
 
+sub select_source ($$) {
+    my ($direction, $n) = @_;
```

Comment:
such functions are really fragile - we have keyround to do that. Just $self->key_round('inst-instsourcedialog-' . get_var('INSTALL_SOURCE'), 'down');